src/multimap_table.rs

```diff
@@ -241,16 +245,22 @@ impl Into<u8> for DynamicCollectionType {
 ///
 /// (when type = 2) root (8 bytes): sub tree root page number
 /// (when type = 2) checksum (16 bytes): sub tree checksum
-#[derive(Debug)]
 #[repr(transparent)]
-pub(crate) struct DynamicCollection {
+pub(crate) struct DynamicCollection<V> {
+    _value_type: PhantomData<V>,
     data: [u8],
```

Comment:
even though `PhantomData` is zero-sized, the inner DST must be placed last. See [Exotically Sized Types](https://doc.rust-lang.org/nomicon/exotic-sizes.html) section of the nomicon for more info

Reply:
Can you add a comment referencing that? I didn't know that was necessary!

Reply:
done!